src/server/controllers/participants.js

```diff
@@ -242,3 +242,18 @@ exports.getCourseDetails = async (req, res) => {
     res.status(500).send('Server Error');
   }
 };
+
+// Add new course for special participant
+exports.addCourse = async (req, res) => {
+  try {
+    const participantId = req.params.id;
+    const { courseData } = req.body;
+    courseData.participant_id = participantId;
+    await courses.create(courseData);
+    res.status(200).send({ message: 'Adding date is done' });
```

Comment:
'Adding date is done' => 'Adding course is done' 😀

Reply:
I'll do it
thanks Tom :smile: